libconnman-qt/networkmanager.cpp

```diff
@@ -1515,9 +1512,25 @@ bool NetworkManager::isValid() const
     return m_priv->m_servicesAvailable && m_priv->m_technologiesAvailable;
 }
 
+NetworkManager::State NetworkManager::globalState() const
+{
+    QString state = m_priv->m_propertiesCache.value(StateProperty).toString();
+    if (state == QStringLiteral("offline")) {
+        return OfflineState;
+    } else if (state == QStringLiteral("idle")) {
+        return IdleState;
+    } else if (state == QStringLiteral("ready")) {
+        return ReadyState;
+    } else if (state == QStringLiteral("online")) {
```

Comment:
style nit: else-after-jump chain in new code

Reply:
As discussed in offline, there are pros and cons, extra lines vs no braces vs this.